resources/views/layouts/app.blade.php

```diff
@@ -33,8 +33,7 @@
 </head>
 <body class="font-sans antialiased bg-white dark:bg-gray-900">
 
-    <div
-        class="bg-linear-to-br from-gradient-light-blue via-gradient-light-pink to-gradient-light-pink dark:from-gradient-dark-blue dark:via-gradient-dark-pink dark:to-gradient-dark-pink">
+    <div class="bg-[#070E1C]">
```

Comment:
```suggestion
    <div class="bg-conference-dark">
```
You can use the colour name from the tailwind config.